websites/management/commands/markdown_cleaning/baseurl_rule.py

```diff
@@ -54,7 +54,7 @@ def replace_match(
         url = urlparse(dest_match.group("dest"))
 
         # This is probably a link with image as title, where the image is a < resource >
-        if R"{{<" in link.text or link.text_links:
+        if R"{{<" in link.text or '![' in link.text:
```

Comment:
This change is unrelated to the rest of this PR, but fixes a failing test. Why was there a failing test?

1. #1174 was branched from master....altered this test case; tests were passing
2. #1178 was branched from master... So it was using the original version of `baseurl_test.py`. A change was made that would break the new version introduced in #1174, but that new version wasn't on this branch, so... all tests pass. 
3. #1174 was merged
4. #1178 was merged without a rebase... had stale master but no conflicts, so merge was allowed. But then we ended up with a failing test on master. 

This fixes the issue.